tests/test_integration/helpers.py

```diff
@@ -0,0 +1,62 @@
+from time import time
+from uuid import uuid4
+
+
+def create_partner(service, **custom_attrs):
+        # can't use a uuid because of 30 char limit:
+        unique_id = str(time()).replace('.', '')
+
+        attrs = {
+            'id': 'WL_TEST-%s' % unique_id,
+            'name': 'TEST',
+            'parent_partner_id': 'WL_YOLA',
```

Comment:
Using `WL_YOLA` here restricts which user/pass you can run the test settings.

Maybe that's okay?

This tripped me up when running.  Suggesting either:
* this line use the client's configured username
* update the test sample to use username `WL_YOLA`